Majasdarbs/02-majasdarbs/Portfolio/alvisfranks.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Alvis Franks | Front-end Developer</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <meta charset="UTF-8" />
+    <style>
+      section {
+        padding: 50px;
+        min-height: 10vh;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+      }
+      body {
+        background-color: #252422;
+        color: #fffcf2;
+        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+          Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+        margin: auto;
+      }
+      h1 {
+        font-weight: 700;
+        font-size: 2rem;
+        font-style: normal;
+      }
+      h2 {
+        font-size: 1.8rem;
+      }
+      a {
+        text-decoration: none;
+        color: #eb5e28;
+        font-style: normal;
+        font-weight: 600;
+        text-transform: uppercase;
+      }
+      header ul {
+        display: flex;
+        padding-right: 20px;
+        flex-direction: row;
+        justify-content: space-between;
+        list-style: none;
+      }
+      #content-holder {
+        width: 1000px;
+        margin: auto;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="content-holder">
+      <header>
+        <nav>
+          <ul>
+            <li><a href="#work-experience">Work experience</a></li>
+            <li><a href="#education">Education</a></li>
+            <li><a href="#skills">Skills</a></li>
+            <li><a href="#faq">FAQ</a></li>
+            <li><a href="#languages">Languages</a></li>
+            <li><a href="#contact-me">Contact info</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+        <!-- ABOUT ME -->
+        <section id="aboutme">
+          <a href="https://github.com/alvisfranks">
+            <svg
```

Comment:
Just a note - you can store svgs in a separate files, the same way you do it with other assets.https://css-tricks.com/using-svg/